docs/controls/Scatter.md

```diff
@@ -0,0 +1,188 @@
+# Scatter
```

Comment:
Can you link this graph in the same place as other graphs so that consumers can get to it better?

Reply:
Added in [Readme.md](https://github.com/cerner/carbon-graphs/pull/128/commits/1d9e855ab0886f8a110f6e0eabb687c4dca4f18f#diff-04c6e90faac2675aa89e2176d2eec7d8R69), also removed it from [potential-feature](https://github.com/cerner/carbon-graphs/pull/128/commits/1d9e855ab0886f8a110f6e0eabb687c4dca4f18f#diff-04c6e90faac2675aa89e2176d2eec7d8L107) as the feature is added now!